ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/dcatdataset/entities/DcatDataset.java

```diff
@@ -0,0 +1,6 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.domain.dcatdataset.entities;
+
+import org.apache.jena.rdf.model.Model;
+
+public record DcatDataset(String id, Model model) {
```

Comment:
why is this id and not collectionName? can a collection have more than one dataset?